### DISCUSSION TITLE
"Issues with GitHub. I made commits in the last 4 days, but the platform did not count them towards my contributions. What should I do?"

### DISCUSSION THREAD

**Discussion by alef-monteiro** [community/community#54520] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [x] Issues
- [ ] Projects

### Body

"Issues with GitHub. I made commits in the last 4 days, but the platform did not count them towards my contributions. What should I do?"

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by tuves** [comment]
Hi @alef-monteiro,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your other posts were removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
